test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -424,6 +424,34 @@ public void Created_WithRelativeUriParameter_SetsCreatedLocation()
         }
 
         [Fact]
+        public void Created_IDisposableObject_RegistersForDispose()
+        {
+            // Arrange
+            var mockHttpContext = new Mock<DefaultHttpContext>();
+            mockHttpContext.Setup(x => x.Response.OnResponseCompleted(It.IsAny<Action<object>>(), It.IsAny<object>()))
+                .Verifiable();
+            var uri = new Uri("/test/url", UriKind.Relative);
+
+            var controller = new TestableController()
+            {
+                ActionContext = new ActionContext(mockHttpContext.Object, new RouteData(), new ActionDescriptor())
+            };
+            var input = new DisposableObject();
+
+            // Act
+            var result = controller.Created(uri, input);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
+            Assert.Equal(StatusCodes.Status201Created, result.StatusCode);
+            Assert.Equal(uri.OriginalString, result.Location);
+            Assert.Same(input, result.Value);
+            mockHttpContext.Verify(
```

Comment:
Hmm, this looks odd to me. I haven't used Moq in a while, but why would this need to repeat the action lambda to verify? I thought that was already set up in the call to `.Setup(....).Verifiable()` earlier?